app/subscriber/package.json

```diff
@@ -18,6 +18,8 @@
     "@react-keycloak/web": "3.4.0",
     "@reduxjs/toolkit": "2.2.5",
     "axios": "1.7.2",
+    "chart.js": "4.4.4",
```

Comment:
Add charts to the subscriber app so that we can dynamically view them when creating a report.